README.md

```diff
@@ -1,80 +1,201 @@
-pupe-pat
-========
+[//]: # (The name of the repository with a brief, high-level description of the
+project)
+# Pupil Plate Ellipticity (PUPE)- Photometric Analysis Tool (PAT)
 
-Pupil Plate Ellipticity (PUPE)- Photometric Analysis Tool (PAT)
----------------------------------------------------------------
+[//]: # (A description of how to install the project on a local machine for
+development)
+
+## Purpose
+The purpose of this tool is to facilitate processing pupil plate
+images collected during the collimation of a telescope. A "pupil plate"
+is a defocused star that is defocused so much that it appears as a donut.
+The shape of that donut can be used to characterize
+(and hopefully be able to eliminate) optical distortion.
 
-### Purpose
-The purpose of this tool is to facilitate processing of pupil plate
-images collected during the collimation of a Telescope.
 The goal of this software tool is to help the user isolate which
 pupil plate image is the best out of a pre-exposed set of images by
-identifying a set of quantitative characteristics processed through
-photometric analysis tools.
-
-At the end of this development the team should have a tool that they
-can use to process many fits file images, quantify the best image to
-move forward with, and visualize how the various tip/tilts of the M2
-mechanism affect various photometric quantities.
-This software should be readily available at 1m sites and should
-process images quickly enough to not burden the team 
-(100 images in 5 minutes).
-
-### Problem
-During collimation engineers use a script to take a number of out
+identifying the best position for the tip and tilt of the M2 mechanism.
+
+## Installation
```

Comment:
Probably best to state which installation method is preferred for those who do not understand the pros and cons of each.
Also, make it clear there are two alternatives "Alternatively, you can ..."

Reply:
Done.